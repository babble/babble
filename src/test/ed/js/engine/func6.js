
a = function b() {
    print( "hi" );
}

a();


a = function b( num ) {
    if( num > 0 ) {
        print( num-- );
        b( num );
    }
}

a(3);

