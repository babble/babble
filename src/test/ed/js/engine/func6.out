hi
3
2
1
ok
3
2
1
